src/PowerShellEditorServices/Services/DebugAdapter/DebugService.cs

```diff
@@ -687,8 +680,20 @@ private async Task FetchGlobalAndScriptVariablesAsync()
             var scopeVariableContainer = new VariableContainerDetails(this.nextVariableId++, "Scope: " + scope);
             this.variables.Add(scopeVariableContainer);
 
-            IReadOnlyList<PSObject> results = await _executionService.ExecutePSCommandAsync<PSObject>(psCommand, CancellationToken.None)
-                .ConfigureAwait(false);
+            IReadOnlyList<PSObject> results;
+            try
+            {
+                results = await _executionService.ExecutePSCommandAsync<PSObject>(psCommand, CancellationToken.None)
+                    .ConfigureAwait(false);
+            }
+            catch (CmdletInvocationException ex)
```

Comment:
Please add a comment as to why this exception is caught and eaten here, except for 'psargumentoutofrangeexception' types.